### PR TITLE
Update docs for beta release

### PR DESCRIPTION
**Pull request by ale7714**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +77,7 @@ For example:
 
    ```sh
    aws s3 sync "s3://my-bucket/coverage/$SHA" coverage/
-   cc-test-reporter sum-coverage --output - coverage/codeclimate.*.json | \
+   cc-test-reporter sum-coverage --output - --parts <number-partials-results> coverage/codeclimate.*.json | \
```
Comment (pbrisbin) [inline README.md]: Can we use some example `$VARIABLE` and explain it after, like we do above for `$N` and `$SHA`?

File: man/cc-test-reporter-format-coverage.1.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ Locate, parse, and re-format supported coverage sources.
 Output to *PATH*. If *-* is given, content will be written to *stdout*. Defaults
 to *coverage/codeclimate.json*.
 
+## -p, --prefix *PATH*
+
+*PATH* of the working directory where the tests were run. Defaults to current
```
Comment (pbrisbin) [inline man/cc-test-reporter-format-coverage.1.md]: How about:

> The prefix to remove from absolute paths in coverage payloads, to make them relative to the project root. This is usually the directory in which the tests were run. Defaults to the current working directory.

Please also update the top-level `SYNOPSIS`

```
 cc-test-reporter-format-coverage [--output=<path>] [--prefix=<PATH>]
```

File: man/cc-test-reporter-sum-coverage.1.md [hunk 3]
```diff
@@ -21,6 +21,10 @@ Combine (sum) multiple pre-formatted coverage payloads into one.
 Output to *PATH*. If *-* is given, content will be written to *stdout*. Defaults
 to *coverage/codeclimate.json*.
 
+## -p, --parts *NUMBER*
+
+*NUMBER* of partial test coverage results. Required flag.
```
Comment (pbrisbin) [inline man/cc-test-reporter-sum-coverage.1.md]: How about

> Expect `$NUMBER` payloads to sum. If this many arguments are not present, `sum-coverage` will error. This ensures you don't accidentally sum incomplete results.

We shouldn't declare when things are optional or required in the individual options description. That should be conveyed via the top-level `SYNOPSIS`:

```
cc-test-reporter-sum-coverage --parts=<number> [--output=<path>] FILE [FILE, ...]
```

**Comment by ale7714** [comment]
Alright i updated the PR :)

File: README.md [hunk 4]
```diff
@@ -16,17 +14,22 @@ machine:
...
 
   post:
-    - ./cc-test-reporter after-build --exit-code $CIRCLE_EXIT_CODE
+    - ./cc-test-reporter after-build --exit-code $TEST_SUITE_EXIT_CODE
 ```
 
+  Where:
+
+  - `$TEST_EXIT_CODE` should be the exit code when the test suite ran. Some CI
+    services don't expose directly a var with the
```
Comment (pbrisbin) [inline README.md]: How about just `$EXIT_CODE`, and

> `$EXIT_CODE` should be the exit code of your test suite process. Some CI system expose this as an environment variable, for others you may need to manually capture `$?` to provide it to `after-build` later. Providing this is optional and will enable future Code Climate features such as pass/fail metrics.

/cc @noahd1

File: man/cc-test-reporter-format-coverage.1.md [hunk 5]
```diff
@@ -21,6 +21,12 @@ Locate, parse, and re-format supported coverage sources.
 Output to *PATH*. If *-* is given, content will be written to *stdout*. Defaults
 to *coverage/codeclimate.json*.
 
+## -p, --prefix *PATH*
+
+The prefix *PATH* to remove from absolute paths in coverage payloads, to make
```
Comment (pbrisbin) [inline man/cc-test-reporter-format-coverage.1.md]: I would just say "The prefix to remove..". No one said you have to use the options argument in its description :), and I think it reads a little awkwardly when you do here.

File: README.md (hunk 4 above)
Comment (noahd1) [inline README.md]: I agree with `EXIT_CODE`? instead.

The documentation is tricky here, and I think we want to provide examples of the two ways I can think of to do it. One is with a provided env var, and the other is capturing the exit code as you mentioned.

* Travis CI: `./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT ` (Note that `$? [should _not_](https://github.com/travis-ci/travis-ci/issues/3771) be used in Travis CI)
* Circle, Jenkins: manual way. Can we cite a (tested) example (our app I guess)? Also, I saw linux `pipefail` could be useful in some cases if you're 1-lining it but guess we don't want to encourage that.

Comment (noahd1) [inline README.md]: I probably wouldn't note it as optional FWIW, or discuss ways in which it may or may not be used.

**Pull request merged**